float32 amount
---
int8 status
